picasso/src/main/java/com/squareup/picasso/Request.java

```diff
@@ -500,6 +497,10 @@ public Request build() {
         throw new IllegalStateException(
             "Center inside requires calling resize with positive width and height.");
       }
+      if (onlyScaleDown && (targetWidth == 0 && targetHeight == 0)) {
```

Comment:
@JakeWharton This method is in the `Request` class, which seems to be in line with the other checks such as for null lists, invalid properties, and center[Crop|Inside] checks.